packages/main/src/components/VariantManagement/index.tsx

```diff
@@ -353,6 +350,9 @@ const VariantManagement = forwardRef((props: VariantManagementPropTypes, ref: Re
             });
           }
           callbackProperties.variants.push({ ...child.props, ...updatedProps, variantItem: currentVariant });
+          if (typeof onSaveManageViews === 'function') {
+            onSaveManageViews(enrichEventWithDetails(e, callbackProperties));
+          }
```

Comment:
Please call the function outside of the `map` and setter.

Reply:
Moved